finality-aleph/src/network/manager/service.rs

```diff
@@ -419,21 +435,30 @@ impl<D: Data> IO<D> {
     ) -> Result<(), Error> {
         let mut maintenance = interval(Duration::from_secs(MAINTENANCE_PERIOD_SECONDS));
         loop {
+            trace!(target: "aleph-network", "Manager Loop started a next iteration");
             tokio::select! {
-                maybe_command = self.commands_from_user.next() => match maybe_command {
+                maybe_command = self.commands_from_user.next() => {
+                    trace!(target: "aleph-network", "Manager Loop command case");
```

Comment:
"Manager received a command from user" would be better, same style in the other `trace`s.

Reply:
Done